tensorbay/exception.py

```diff
@@ -162,19 +185,22 @@ def __init__(  # pylint: disable=super-init-not-called
         resource: Optional[str] = None,
         identification: Union[int, str, None] = None,
     ) -> None:
-        pass
+        if response is not None:
+            super().__init__(response)
+            return
 
-    def __str__(self) -> str:
-        pass
+        self._resource = resource
+        self._identification = identification
```

Comment:
Can we use `name` instead of `identification`?

Reply:
The `identification` parameter includes not only names but also draft numbers, revisions.